tools/cloud-workstations/configure-hpc-toolkit.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# Check if the hpc-toolkit folder exists in the user home directory
+if [ ! -d "$HOME/hpc-toolkit" ]; then
+	# If not, clone the repository
+	git clone https://github.com/GoogleCloudPlatform/hpc-toolkit.git "$HOME/hpc-toolkit"
+	cd "$HOME/hpc-toolkit" || exit
+	make install-dev-deps
+	pre-commit install
```

Comment:
Why not also `make` & `make install`?

Reply:
I was trying to strike a balance between time an instance takes to create and how much of it would be needed anyway since people maybe do a git pull. 
I think that since this is created by the workstation, it is very likely someone would need a make. 
personally, I dislike make install, so I will leave it out.

Reply:
PTAL